doc/man3/OSSL_PARAM.pod

```diff
@@ -40,11 +40,11 @@ suitable form for the internal structure of the object.
 
 =item * Request parameters of some object
 
-The caller (the I<requestor>) sets up the B<OSSL_PARAM> array and
+The caller (the I<requester>) sets up the B<OSSL_PARAM> array and
```

Comment:
Hmm, interesting.  This might be a difference of opinion; I think I've somewhere seen a claim that "requestor" was more British than American English, but can't find that reference.  Googling now, though, I see numerous pages talking about the difference between people ("requester") and computer programs ("requestor"), for example https://english.stackexchange.com/questions/29254/whats-the-difference-between-requester-and-requestor

Reply:
Note that codspell relies on mainstream dictionaries:
* Both are missing from [Oxford Learner's Dictionary](https://www.oxfordlearnersdictionaries.com/spellcheck/english/?q=requester)
* [Cambridge](https://dictionary.cambridge.org/dictionary/english/requester?q=requestor) suggests **_requester_**, but _requestor_ is also acceptable.
* [Collins](https://www.collinsdictionary.com/dictionary/english/requestor) suggests **_requester_** too, but _requestor_ is "a rare spelling of _[requester](https://www.collinsdictionary.com/dictionary/english/request)_".
* [Merriam-Webster](https://www.merriam-webster.com/dictionary/requestor) knows only **_requester_**.

Usage may vary, but Google Ngram Viewer tends to agree it is [_requester_ over _requestor_](https://books.google.com/ngrams/graph?content=requestor%2Crequester&year_start=1800&year_end=2019&corpus=en-2019&smoothing=3):

![requestor](https://github.com/openssl/openssl/assets/3234522/4e8b2567-1bab-4e50-87e3-e83b6e21b377)

Reply:
That said, I am OK to revert to requestor based on your link. I am not a native speaker.

Reply:
In any case, the _requestor_ → _requester_ fix has been removed from this PR and moved to  #20924 together with all **`doc/man*`** fixes.